app/components/ProductCard/index.js

```diff
@@ -3,137 +3,206 @@
+            <Typography>{this.props.product.display_name}</Typography>
+        </div>
+    )
+
+    renderRating = () => {
+        const count = dataChecking(this.props.product, 'review', 'count');
+        const rating = dataChecking(this.props.product, 'review', 'rating');
+        const rate = Math.round(rating / 2);
+        const rateArr = [];
+
+        if (count < 0 && rating < 0) {
             return null;
         }
+
+        for (let i = 0; i < 5; i++) {
+            if ((i + 1) <= rate) {
+                rateArr.push(<Star key={`${this.props.product.id}-${i}`} style={{ color: '#FFD700' }} />);
+            } else {
+                rateArr.push(<StarBorder key={`${this.props.product.id}-${i}`} style={{ color: '#FFD700' }} />);
+            }
+        }
+
         return (
-            <div onClick={() => this.props.deleteFromWishlist()}>
-                <i className="fa fa-times cross-icon" aria-hidden="true"></i>
+            <div style={{ display: 'flex', flexDirection: 'row' }} className="product-rating">
+                <div style={{ justifyContent: 'flex-start', display: 'flex', flexDirection: 'row' }}>
+                    {rateArr}
+                </div>
+                <Typography variant="overline" style={{ marginLeft: 5, alignSelf: 'center' }}> ({count})</Typography>
             </div>
         );
     }
 
     render() {
         return (
-            <div
-                className={`product-card ${this.props.listViewMode ? 'list-view-card' : 'grid-view-card'}`}
-                key={this.props.product.id}
-            >
-                <div className={`product-card-content ${!this.props.product.instock ? '' : 'OOS'}`}>
-                    <div className="product-card-images">
-                        {this.props.allowDelete &&
-                            <div onClick={() => this.props.deleteFromWishlist()}>
-                                <i className="fa fa-times cross-icon" aria-hidden="true"></i>
-                            </div>
-                        }
-                        {this.renderFeatures()}
-                        <NavLink to={this.props.url}>
-                            {
-                                !this.props.product.instock &&
-                                    <div>
-                                        <span className="OOS-word">Out Of Stock</span>
-                                    </div>
-                            }
-                            {
-                                dataChecking(this.props.product, 'image', 'medium') ?
-                                    <div className="product-card-img">
-                                        <img src={this.props.product.image.medium} alt="product.jpg" className="product-img" />
-                                    </div>
-                                    :
-                                    null
+            <Card style={{ boxShadow: 'none' }} className="product-container">
```

Comment:
@limsukjing I thought you have refine for product cards layout, right? haven't push latest code?